sklearn/feature_extraction/text.py

```diff
@@ -1519,6 +1519,7 @@ class TfidfTransformer(_OneToOneFeatureMixin, TransformerMixin, BaseEstimator):
           been applied.
         - 'l1': Sum of absolute values of vector elements is 1.
           See :func:`preprocessing.normalize`.
+        - None: The program does not normalize the vectors at the end.
```

Comment:
None should also be listed in the parameter description on line 1514:
``norm: {'l1', 'l2'} or None, default='l2'``

Also I would not put None in this bullet list, since the list describes `Each output row will have unit norm, either:`. I would just put that after the list, and maybe rephrase a bit: something like `If None, output rows are not normalized.`